.travis.yml

```diff
@@ -24,15 +24,15 @@ env:
     - COVERALLS=false
     - NOSE_FLAGS="--processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50"
     - NOSE_TEST_LIST="analysis auxiliary coordinates core formats topology utils"
-    - PYTEST_FLAGS="--disable-pytest-warnings"
+    - PYTEST_FLAGS="--disable-pytest-warnings -n 2"
```

Comment:
@kain88-de That doesn't work. I don't think there is anything like `--numprocesses`